app/styles/app.css

```diff
@@ -38,7 +38,8 @@ html, body {
   text-align: center;
   vertical-align: middle;
   width: 100%;
-  position: fixed;
+  position: relative;
```

Comment:
Your code is correct!
We can however reduce the code size further!
* no need to mention relative positioning, that's the default
* just add a `bottom: 0`
* and remove the `vertical-align: center` _as this is not an inline element, so it is not of much use_ 😅 
* remove the `height: 40px` and instead use a `margin-bottom`
* just use `rem` instead of `px`

Reply:
I have updated the PR. Thanks for the suggestions !

Reply:
Great 😄 👍🏾